subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -90,6 +104,55 @@ static int update_trigger_cb(u16_t obj_inst_id)
 #endif
 }
 
+void *binding_pre_write(u16_t obj_inst_id, size_t *data_len)
+{
+	*data_len = TRANSPORT_BINDING_LEN;
+	return binding_mode_validation_buf;
+}
+
+static int binding_validate(u16_t obj_inst_id, u8_t *data, u16_t data_len,
+			    bool last_block, size_t total_size)
+{
+	u16_t mode, len;
+
+	for (mode = BINDING_MODE_U; mode < BINDING_MODE_UNKNOWN; mode++) {
+		len = strlen(binding_modes[mode]);
```

Comment:
Do we need this len var?  I think we could check strlen(binding_modes[mode]) directly below.